live/index.php

```diff
@@ -88,51 +88,53 @@ function gtag() { dataLayer.push(arguments); }
     <div id="header"></div>
 
     <div class="header_padding" id="live-stream-wrapper">
-        <div id="live-objects" class="center">
-            <h1><span style="color: red; font-weight: bold;">LIVE</span> AT CORNERSTONE</h1>
-            <h2>9:00am | 11:00am | 6:00pm ET</h2>
-            <div id="live-iframe">
-                <iframe id="live-stream" class="drop-shadow"
-                    src='https://www.youtube.com/embed/live_stream?channel=<?php echo $channelId ?>' frameborder="0"
+        <h1><span style="color: red; font-weight: bold;">LIVE</span> AT CORNERSTONE</h1>
+        <h2>9:00am | 11:00am | 6:00pm ET</h2>
+        <div id="live-iframe">
+            <div id="resize-wrapper">
+                <iframe class="drop-shadow"
+                    src='https://www.youtube.com/embed/live_stream?channel=UCSJ4gkVC6NrvII8umztf0Ow' frameborder="0"
```

Comment:
Is there a reason why this was changed from a dynamic variable fetched by php to static?

Reply:
Not sure how that got there, sorry! Committed reversion.